packages/wonder-blocks-tooltip/components/tooltip-bubble.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import {css, StyleSheet} from "aphrodite";
+import * as React from "react";
+import {Popper} from "react-popper";
+
+import visibilityModifierDefaultConfig from "../util/visibility-modifier.js";
+
+import type {PopperChildrenProps} from "react-popper";
+import type {Placement} from "../util/types.js";
+
+type Props = {|
+    // The content to be shown in the bubble.
+    // TODO(somewhatabstract): Make this the TooltipContent type.
+    children: string,
+
+    // The element that anchors the tooltip bubble.
+    // This is used to position the bubble.
+    anchorElement: ?HTMLElement,
+
+    // Where should the bubble try to go with respect to its anchor.
+    placement: Placement,
+|};
+
+export default class TooltipBubble extends React.Component<Props> {
+    _renderContent(childrenProps: PopperChildrenProps) {
+        const {children} = this.props;
+        const {
+            ref,
+            outOfBoundaries,
+            style,
+            placement,
+            //  arrowProps,
```

Comment:
Will `arrowProps` be making a comeback in a subsequent PR?

Reply:
Yes, in fact, it's in PR #155